core/storage/base_model/gae_models.py

```diff
@@ -1115,6 +1115,13 @@ class BaseSnapshotMetadataModel(BaseModel):
     # snapshot content model.
     content_user_ids = ndb.StringProperty(repeated=True, indexed=True)
 
+    @staticmethod
+    def get_deletion_policy():
+        """Collection rights are deleted only if the corresponding collection
```

Comment:
Why a collection-specific reference in a "base" model?

Also, not sure how this translates to "locally pseudonymize". Probably needs more detail.